### DISCUSSION TITLE
Can’t purchase/restore

### DISCUSSION THREAD

**Discussion by swiftbird07** [MobiusSync/MobiusSync#109] [Q&A]
Only on iOS 16.4 I get the warning "Can't sync [...] Syncthing has been disabled.
I thought maybe its because of the 20 MB limit, but there is another bug that nothing happens if I click on "Purchase".

OS: iOS 16.4
Model: iPhone 13 Pro Max
Syncthing Version: Newest from AppStore

**Comment by SimonPickup** [comment]
Thanks for reporting.

Yes, that message is because of the 20MB limit.

We thought we had fixed a bug with the Purchase button not working #46 in the latest release. This was happening for users who had set different font sizes. Do you have any non-default settings like that?

In any case, the easy workaround is to delete **Möbius Sync** and buy **Möbius Sync Pro** instead (at the same price as in-app purchase). You will have to set up your shares again.

**Reply by swiftbird07** [reply]
I had one text size smaller set, but changing back to  normal size (see screenshot) didn't help. I already purchased in-App on my iPad (where it worked with no problem) and I just want to restore the payment for my Apple-ID and I don’t see paying twice for the same app. That seems unfair to me.

Did you test the App for 16.4? Because there is a difference between #46 as I don’t see the pop-up to purchase at all.

![IMG_9246](https://user-images.githubusercontent.com/48106483/229342108-7df805f2-cbd9-4d8f-b269-8822888cb5b8.png)

**Reply by swiftbird07** [reply]
Hello?

**Reply by SimonPickup** [reply]
Yes, it has been tested on 16.4. I also just tried a non-English language in case that was the problem. Do you have any non-default settings, e.g. security, networking, VPN, accessibility settings, or settings differences between iPad and iPhone that I can try to reproduce? (I also used exact model you have - iPhone 13 Pro Max).

We have no ability to issue refunds, but you can try requesting a refund on the in-app purchase here: https://reportaproblem.apple.com/
